src/Bundle/Context/Initiator/LegacyRequestContextInitiator.php

```diff
@@ -27,7 +28,17 @@ public function __construct(
         private RegistryInterface $resourceRegistry,
         private RequestConfigurationFactoryInterface $requestConfigurationFactory,
         private RequestContextInitiatorInterface $decorated,
+        private ?VarsResolverInterface $varsResolver = null,
     ) {
+        if (null === $varsResolver) {
+            trigger_deprecation(
+                'sylius/resource-bundle',
+                '1.10',
```

Comment:
Just checking, isn't current version is 1.13?

```suggestion
                '1.13',
```